wordpress-mobile/WordPressCom-Stats-iOS
# Move non-periodic sections to Insights

Closes #257 

Moved all of the non-periodic sections from the Days/Weeks/Months/Years screens to the Insights screen. This includes comments, tags/categories, followers and publicize.

I also had to change the services for batching to move the calls for those sections into the batched call for Insights. While doing this I realized that the calls for these sections don't really need/use data or period type parameters so those got removed as well.

There is still quite a bit of duplicated logic between all of the view controllers and #223 will eventually address that.

__Things To Test:__
* Make sure tapping on the rows works properly - expanding nested rows, View All on sections that have it, launching out to Safari on a tag or post.
* Pull to refresh to make sure data is consistent.
* Rotation.
* iPad and iPhone consistency.
* Make sure data matches (quick check) to web stats.

__How To Test:__
* Check out the project and launch StatsDemo after filling in an OAuth2 token, site ID and time zone in `WPSViewController`. or
* Ask Aaron nicely to deploy a new StatsDemo internal build based upon this branch/PR. :smile: 

Needs Review: @isaackeyet, @jancavan, @aerych, @jleandroperez
